framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -947,18 +965,47 @@ private AnnotationMirror createNumberAnnotationMirror(List<Number> values) {
                 "ValueAnnotatedTypeFactory: unexpected class: " + first.getClass());
     }
 
+    /**
+     * Returns the set of possible values. Returns the empty list if no values are possible (for
+     * dead code). Returns null if any value is possible -- that is, if no estimate can be made --
+     * and this includes when there is no constant-value annotation so the argument is null.
+     */
     public static List<Long> getIntValues(AnnotationMirror intAnno) {
```

Comment:
Because it supports the `@IntVal` annotation.
I've added documentation of this, and I have documented all the parameters.